Makefile

```
@@ -127,7 +127,7 @@ execute-e2e-from-within-container:
 	NODE_ENV=test LOGLEVEL=error $(BIN_DIR)/jest --config ./test/jest-e2e.config.js --bail --runInBand --ci --reporters=default --reporters=jest-junit
```

Small nit, this might be possible

```suggestion
	make execute-e2e-from-within-container-cached
```